source/docs/casper/concepts/design/rewards.md

```diff
@@ -30,21 +30,23 @@ Rewards are divided into these categories:
 - **Block rewards**: These rewards are received for each proposed block that is finalized. They incentivize timely participation in building the chain.
 - **Finality signature rewards**: These rewards are received for collecting finality signatures for each block and generating a finality signature to sign a block. They incentivize the creation, propagation and publication of finality signatures, which is critical in establishing common knowledge of the canonical chain.
 
-In each round, a total reward pool is shared among all participating validators proportionally to their weight. The `round_seigniorage_rate` setting in the chainspec determines the reward pool as a fraction of the total supply. This value is calculated based on the total supply, target inflation rate, and minimum round length.
+In each round, a total reward pool is shared among all participating validators proportionally to their weight, as long as all validators fully participate in the processes that are rewarded by this mechanism. These processes are block creation, signature creation, signature propagation and signature publication as part of block proposals. 
 
-Each switch block triggers a reward calculation. To account for potential network lag delaying the timely arrival of signatures for finalized blocks, the calculation "looks back" into the previous eras. The number of eras to look back into is specified using the `signature_rewards_max_delay` setting in the chainspec. The rewards formula is additive and calculates rewards over two or more switch blocks.
+The total reward pool for a block is determined by the `round_seigniorage_rate` setting in the chainspec. This value, together with current total supply and minimum round length, is used to compute the full allocation of rewards for a particular block. The rate itself is set to result in a given target annual inflation, provided that validators fully participate in the rewardable processes described above.
 
-Blocks carry information on their proposer and the finality signatures collected for several past blocks. Global state contains data on token supply and validator weights as part of the Mint and Auction states. Based on these inputs, the rewards are calculated according to a formula. Rewards are designed to be proportional to weight on average, as long as blocks are created and the finality signatures are propagated and published in a timely manner. <!-- TODO Link to the formula in the CEP or elsewhere -->
+Each switch block triggers a reward calculation. To account for potential network lag delaying the timely arrival of signatures for finalized blocks, the calculation "looks back" into previous eras. The number of prior blocks to look up is specified using the `signature_rewards_max_delay` setting in the chainspec.
```

Comment:
Since we are talking about switch blocks here, emphasize that switch blocks can be rewarded now.

```suggestion
Each switch block triggers a reward calculation. To account for potential network lag delaying the timely arrival of signatures for finalized blocks, the calculation "looks back" into previous eras. In particular, this enables rewards for switch blocks, which was impossible with the prior Highway-specific calculation. The number of prior blocks to look up is specified using the `signature_rewards_max_delay` setting in the chainspec.
```